composer/algorithms/dropblock/dropblock.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+import logging
+from dataclasses import asdict, dataclass
+from functools import reduce
+from operator import mul
+from typing import Optional
+
+import torch
+import yahp as hp
+from torch.nn import functional as F
+
+from composer.algorithms import AlgorithmHparams
+from composer.core.types import Algorithm, Event, Logger, State, Tensor
+
+log = logging.getLogger(__name__)
+
+
+def dropblock(x: Tensor, drop_prob: float = 0.1, block_size: int = 7, batchwise: bool = True):
+    '''
+    See :class:`DropBlock`.
+
+    Args:
+        drop_prob (float): Drop probability
+        block_size (int): Size of blocks to drop out
+        batchwise (bool): Whether to mask per batch (faster) or per sample
```

Comment:
Can you add the default values to the docstring, and also include a line for what is being returned?

Reply:
Done